test/widget_test.dart

```diff
@@ -15,16 +15,16 @@ void main() {
     // Build our app and trigger a frame.
```

Comment:
The test description no longer matches what actually is being tested, so that should be changed.

```suggestion
'Sample test'
```
However as another note, this test only serves as a dummy test as it should be expanded should it actually test something properly. I suggest we'll just let it be for the time being a come by and clean up at a later point.

Reply:
Changed description & added more comments for the test.